packages/common/src/types.ts

```diff
@@ -196,6 +201,22 @@ export type HardforkConfig = {
   name: string
   eips?: number[]
   consensus?: ConsensusConfig
+  // Type to optionally override params
+  gasConfig?: {
+    [key: string]: number | bigint | null
+  }
+  gasPrices?: {
+    [key: string]: number | bigint | null
+  }
+  pow?: {
+    [key: string]: number | bigint | null
+  }
+  sharding?: {
+    [key: string]: number | bigint | null
+  }
+  vm?: {
+    [key: string]: number | bigint | null
+  }
 } & EIPOrHFConfig
```

Comment:
I think you don't need to do this, this is already in through `&EIPOrHFConfig`

Reply:
Hey, you are right, I think this got introduced in #3500